mlflow/pyfunc/model.py

```diff
@@ -196,11 +196,12 @@ def _save_model_with_class_artifacts_params(
     :param python_model: An instance of a subclass of :class:`~PythonModel`. ``python_model``
                         defines how the model loads artifacts and how it performs inference.
     :param artifacts: A dictionary containing ``<name, artifact_uri>`` entries.
-                      Remote artifact URIs
-                      are resolved to absolute filesystem paths, producing a dictionary of
-                      ``<name, absolute_path>`` entries. ``python_model`` can reference these
-                      resolved entries as the ``artifacts`` property of the ``context``
-                      attribute. If ``None``, no artifacts are added to the model.
+                      Remote artifact URIs are resolved to absolute filesystem paths, producing
+                      a dictionary of ``<name, absolute_path>`` entries. ``python_model`` can
+                      reference these resolved entries as the ``artifacts`` property of the
+                      ``context`` attribute. If ``None``, no artifacts are added to the model.
+                      If ``<snapshot, huggingface_snapshot_location>`` is provided, then the model
+                      will be fetched from `huggingface_snapshot_location` directly.
```

Comment:
Is this specific to huggingface?

Reply:
Are there any flavors where the snapshot makes sense?

Reply:
Emmm from the implementation it's not specific to huggingface

Reply:
Maybe I'll change it to snapshot_location just in case there're other usages